packages/marshal/src/marshal-justin.js

```diff
@@ -402,6 +403,7 @@ const decodeToJustin = (encoding, shouldIndent = false) => {
       } else {
         out.open('{');
         for (const name of names) {
+          // @ts-ignore Type 'symbol' cannot be used as an index type.ts(2538)
```

Comment:
Interesting. Without the `@ts-ignore`, were you not getting the error before this PR but getting it with these changes? Do you know why?

Reply:
I addressed this issue in #1043 (the new base of this PR) by addressing why the `names` had the wrong types in the first place.